app/views/organizations/members/index.html.erb

```diff
@@ -14,30 +14,30 @@
 
 <%= render CardComponent.new do |c| %>
   <%= c.head do %>
-		<%= c.title t("organizations.show.members"), icon: "organizations", count: @memberships_count %>
+    <%= c.title t("organizations.show.members"), icon: "organizations", count: @memberships_count %>
     <% if policy(@organization).invite_member? %>
       <%= render ButtonComponent.new t(".invite"), new_organization_membership_path(@organization), type: :link %>
     <% end %>
   <% end %>
-    <%= c.divided_list do %>
-		<% @memberships.each do |membership| %>
-			<%= c.list_item_to(
-				policy(@organization).list_memberships? ? edit_organization_membership_path(@organization, membership) : profile_path(membership.user),
-				title: membership.user.handle,
-			) do %>
-				<div class="flex flex-col w-full justify-between">
-					<div class="flex flex-row w-full items-center justify-between">
-						<p class="text-neutral-800 dark:text-white"><%= membership.user.handle %></p>
-						<% if policy(@organization).list_memberships? %>
-							<% if membership.confirmed? %>
-							<p class="text-neutral-500 capitalize"><%= membership.role %></p>
-							<% else %>
-							<p class="text-neutral-500 capitalize"><%= t(".pending") %></p>
-							<% end %>
-						<% end %>
-					</div>
-				</div>
-			<% end %>
-		<% end %>
+  <%= c.divided_list do %>
+    <% @memberships.each do |membership| %>
+      <%= c.list_item_to(
+        policy(@organization).list_memberships? ? edit_organization_membership_path(@organization, membership) : profile_path(membership.user),
+        title: membership.user.handle,
+      ) do %>
+        <div class="flex flex-col w-full justify-between">
+            <div class="flex flex-row w-full items-center justify-between">
+            <p class="text-neutral-800 dark:text-white"><%= membership.user.handle %></p>
+            <% if policy(@organization).list_memberships? %>
+              <% if membership.confirmed? %>
+                <p class="text-neutral-500 capitalize"><%= membership.role %></p>
+              <% else %>
+                <p class="text-neutral-500 capitalize"><%= t(".pending") %></p>
+              <% end %>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+    <% end %>
```

Comment:
These are whitespace changes which I fixed along the way (primarily making everything spaces instead of a mix of tabs and spaces). I can revert the changes if our priority is a small diff, but I thought establishing consistency was a long-term improvement.

Reply:
I prefer to keep formatting changes in separate commit at least whenever possible for easier review. If not, there's still this option for review in UI.

<img width="340" height="313" alt="image" src="https://github.com/user-attachments/assets/e1e195d0-5bda-4700-938e-15d7ef234d94" />

Reply:
No plan to finalize this in two commits?

Reply:
In case there are 2 valuable commits we do rebase to keep them. IMHO still worth it.

Reply:
Done